array-manipulation-1/src/initial.js

```diff
@@ -0,0 +1,8 @@
+/* exported initial */
+function initial(array) {
+  var container = [];
+  for (var i = 1; i < array.length; i++) {
+    container.push(array[i - 1]);
```

Comment:
Instead of doing this, you should start the loop at `0` and end one before the last index.